src/main/java/mil/dds/anet/utils/Utils.java

```diff
@@ -207,9 +216,45 @@ public static String sanitizeHtml(String input) {
     if (input == null) {
       return null;
     }
-    return POLICY_DEFINITION.sanitize(input);
+    return HTML_POLICY_DEFINITION.sanitize(input);
   }
 
+  public static String sanitizeJSON(String inputJson) throws JsonProcessingException {
```

Comment:
Rename method to `sanitizeJson`.
Add a `testSanitizeJson` to `src/test/java/mil/dds/anet/utils/UtilsTest.java`.